internal/gui/keybindings.go

```diff
@@ -82,9 +93,18 @@ func (g *Gui) selectPage(row, col int) string {
 	case 0:
 		p = `trips`
 	case 1:
-		p = `people`
+		p = `cavers`
 	case 2:
-		p = `locations`
+		p = `caves`
 	}
 	return p
-}
\ No newline at end of file
+}
+
+/*
+func (g *Gui) nextPage() {
```

Comment:
See above comment regarding tabbing through pages.

Reply:
This will likely be a larger issue than this PR. So it might be best for the moment to allow the code to stay albeit commented out.